packages/theme-wizard-website/public/templates/collage/variation.html

```diff
@@ -0,0 +1,154 @@
+<!doctype html>
+<html lang="nl-NL">
```

Comment:
```suggestion
<html lang="en">
```

Aangezien alle content in het engels is.

Reply:
Engels en NL door elkaar heen in de collage variation. Heb samples uit NL Design system gekopieerd wat NL is, maar bijv de beschrijvingen erbij in het EN.

Volledig Engels doen of volledig Nederlands?